tests/benchmark_spec_test.py

```diff
@@ -132,3 +137,46 @@ def testBadParameter(self):
     spec = benchmark_spec.BenchmarkSpec(config, NAME, UID)
     with self.assertRaises(errors.Config.UnrecognizedOption):
       spec.ConstructVirtualMachines()
+
+
+class BenchmarkSupportTestCase(unittest.TestCase):
+
+  def setUp(self):
+    # Reset the current benchmark spec.
+    self.addCleanup(context.SetThreadBenchmarkSpec, None)
+
+  def createBenchmarkSpec(self, config, benchmark):
+    try:
+      spec = benchmark_spec.BenchmarkSpec(config, benchmark, UID)
```

Comment:
just fyi - theres also an "assertRaises" that you can use so you don't have to try: return True except: return False